src/ripple/app/misc/HashRouter.h

```diff
@@ -159,12 +160,10 @@ class HashRouter
     std::mutex mutable mMutex;
 
     // Stores all suppressed hashes and their expiration time
-    hash_map <uint256, Entry> mSuppressionMap;
+    beast::aged_unordered_map<uint256, Entry, clock_type,
```

Comment:
Either it can not, or I'm doing something wrong. The constructor fails with another `Stopwatch` as the parameter.
```
1>C:\Dev\rippled-merge\src\ripple/app/misc/HashRouter.h(129): error C2664: 'beast::detail::aged_unordered_container<false,true,ripple::uint256,ripple::HashRouter::Entry,ripple::Stopwatch,ripple::basic_hardened_hash<HashAlgorithm,false>,std::equal_to<Key>,std::allocator<std::pair<const Key,T>>>::aged_unordered_container(const beast::detail::aged_unordered_container<false,true,Key,T,ripple::Stopwatch,ripple::basic_hardened_hash<HashAlgorithm,false>,std::equal_to<Key>,std::allocator<std::pair<const Key,T>>> &)': cannot convert argument 1 from 'ripple::Stopwatch' to 'beast::abstract_clock<Clock> &'
1>          with
1>          [
1>              HashAlgorithm=beast::xxhasher,
1>              Key=ripple::uint256,
1>              T=ripple::HashRouter::Entry
1>          ]
1>          and
1>          [
1>              Clock=ripple::Stopwatch
1>          ] (compiling source file ..\..\src\ripple\unity\app_tx.cpp)
```
I think it's because `Stopwatch` is already a `beast::abstract_clock`, so declaring it with that as the template parameter leaves the constructor expecting an `abtstract_clock<abstract_clock<something>>` and it all falls apart.